Image to image (#370)
huggingface/hub-docs: tasks/src/image-to-image/about.md

## Use Cases

### Style transfer

One of the most popular use cases of image to image is the style transfer. Style transfer models can convert a regular photography into a painting in the style of a famous painter.

> style of Claude Monet.

Reviewer: ```suggestion
```
We can remove this given we fixed above paragraph 🙂

## Task Variants

### Image inpainting

> Image inpaining is widely used during photography editing to remove unwanted objects, such as poles, wires or sensor

Reviewer: ```suggestion
Image inpainting is widely used during photography editing to remove unwanted objects, such as poles, wires or sensor
```
nit :)

## Use Cases

### Style transfer

> One of the first and most popular applications, style transfer can convert a regular photography into a paining in the

Reviewer: ```suggestion
One of the most popular use cases of image to image is the style transfer. Style transfer models can convert a regular photography into a painting in the style of a famous painter.
```

style of Claude Monet.

### Image segementation

Image segmentation represents is a pixel level classification - each pixel is assigned a numeric label, according to
the object it belongs to.

### Image inpainting

> Image inpaining is widely used during photography editing to remove unwanted objects, such as poles, wires or sensor

Reviewer: I'm a bit inclined to have Image Inpainting as a sub-task in task variants, what do you think? 🙂 I feel like other use cases are general enough to be in task variants as well, except for Style Transfer.

Author: Considering the broadness of image to image translation, we can have many interpretations of what is a variant, and what really is the core application :). But I agree all are very broad applications.
